server/models/procedures.sql

```diff
@@ -47,4 +47,10 @@ SOURCE server/models/procedures/trial_balance.sql
 */
 SOURCE server/models/procedures/stock.sql
 
+
+/*
+  role management procedures
+*/
+SOURCE server/models/procedures/roles.sql > /dev/null
```

Comment:
The `> /dev/null` is a bash command, but this is run in SQL.  I'm surprised it works!  Can you remove it?